packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -18,12 +18,15 @@
   "l1BlockTime": 6,
   "baseFeeVaultRecipient": "0x14dC79964da2C08b23698B3D3cc7Ca32193d9955",
   "l1FeeVaultRecipient": "0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f",
+  "operatorFeeVaultRecipient": "0x23618e81E3f5cdF7f54C3d65f7FBc0aBf5B21E8f",
```

Comment:
Should be able to remove this diff per https://github.com/ethereum-optimism/optimism/pull/12166/files#r1941779240